### PR TITLE
Update sample apps for Python SDK v5

### PR DESCRIPTION
**Pull request by tribble**
Update all of the flask examples to work with v5 of the SDK.

I touched up the Directory Sync app a bit to add more paths that exercise more SDK paths. For the other example apps, I just made enough changes to keep them functional and not introduce type errors when passing arguments to the SDK.

### CODE REVIEW

File: python-flask-admin-portal-example/requirements.txt [hunk 1]
```diff
@@ -1,14 +1,14 @@
 certifi==2021.5.30
 charset-normalizer==2.0.6
 click==8.0.1
-Flask==2.0.1
+Flask==2.0.3
```
Comment (tribble) [inline python-flask-admin-portal-example/requirements.txt:4]: Different examples used different flask versions. I set them all to the same version.

File: python-flask-directory-sync-example/requirements.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
-urllib3==1.26.7
+Flask==2.0.3
+workos==5.0.0b1
+urllib3>=2
 python-dotenv
 flask_socketio
-flask-lucide==0.2.0
\ No newline at end of file
+flask-lucide==0.2.0
+Werkzeug==2.0.1
```
Comment (tribble) [inline python-flask-directory-sync-example/requirements.txt:7]: Starting flask would error without this dependency

File: python-flask-admin-portal-example/requirements.txt (hunk 1 above)
Comment (mattgd) [reply python-flask-admin-portal-example/requirements.txt:4]: Any reason we can't bump to the latest Flask version? Also fine with keeping this out of scope.

File: python-flask-admin-portal-example/app.py [hunk 3]
```diff
@@ -13,9 +12,12 @@
 lucide = Lucide(app)
 
 # WorkOS Setup
-workos.api_key = os.getenv("WORKOS_API_KEY")
-workos.project_id = os.getenv("WORKOS_CLIENT_ID")
-workos.base_api_url = "http://localhost:7000/" if DEBUG else workos.base_api_url
+base_api_url = "http://localhost:7000/" if DEBUG else None
```
Comment (mattgd) [inline python-flask-admin-portal-example/app.py:15]: This feels kind of odd to me for a public example. I wonder if we should just use the `WORKOS_BASE_URL` environment variable here.

**Review by mattgd — APPROVED** [review]

File: python-flask-admin-portal-example/app.py (hunk 3 above)
Comment (tribble) [reply python-flask-admin-portal-example/app.py:15]: Agreed. This is a simple change I can make to all of the sample apps.

**Pull request merged**